src/sidebars/favourites/tearout.js

```diff
@@ -16,6 +16,7 @@
 
                         var windowService = window.windowService;
                         var tearoutWindow = windowService.createTearoutWindow(window.name);
+                        var me = {};
```

Comment:
That's mostly a hangover from the original tearout example, but not something that's changed in this PR.